JustFakeIt/MatchingStrategy/BodyMatching/AbsoluteBodyMatching.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace JustFakeIt
+{
+    public class AbsoluteBodyMatching : IBodyMatchingOption
+    {
+        public bool MatchBody(string expected, string actualBody)
+        {
+            return string.IsNullOrEmpty(expected) || actualBody.Equals(expected);
```

Comment:
This is going to throw a wobbly if `actualBody` is null. I don't *think* that can ever happen, but probably good to check.